packages/plugin-core/src/constants.ts

```diff
@@ -640,6 +640,18 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     docLink: "dendron.topic.commands.md",
     docPreview: `![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/images/workbench.help.gif)`,
   },
+  SHOW_NOTE_GRAPH: {
+    key: "dendron.showNoteGraph",
+    title: `${CMD_PREFIX} Show Note Graph`,
```

Comment:
would currently add a `V2` suffix to this command so that people can use this in conjunction with `markdown links` while we're iterating